spec/handlers/newflow/educator_signup/sheerid_webhook_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+module Newflow
+  module EducatorSignup
+    RSpec.describe SheeridWebhook, type: :handler do
```

Comment:
is testing the response handling somewhere too?

Reply:
Yeah. This? https://github.com/openstax/accounts/pull/859/files#diff-82ae3922ca1f3ca175760ea679b5e3f9R170